plugins/woocommerce/client/admin/client/customize-store/assembler-hub/block-editor-container.tsx

```diff
@@ -186,6 +186,7 @@ export const BlockEditorContainer = () => {
 		<BlockEditor
 			renderedBlocks={ blocks }
 			isScrollable={ isScrollable }
+			// @ts-expect-error The type is not up to date.
```

Comment:
Sorry to be nitpicky here, but  the `BlockEditor` component is actually an internal component we can update ourselves.
I was thinking of adding the above comment here: https://github.com/woocommerce/woocommerce/blob/816aee20f4e6eea96ff60724c043b8efa7c3c1bf/plugins/woocommerce/client/admin/client/customize-store/assembler-hub/block-preview.tsx#L49
As the `BlockEditorProvider` is imported from the GB package.

Reply:
No worries! I appreciated it! I'm dealing across multiple TS errors in different branches, and it creates some confusion to me :D 

Fixed with 95ef4cdca179a935189e292cf46d315bb477955e